lib/src/rules.dart

```diff
@@ -269,6 +270,7 @@ void registerLintRules({bool inTestMode = false}) {
     ..register(LeadingNewlinesInMultilineStrings())
     ..register(LibraryNames())
     ..register(LibraryPrefixes())
+    ..register(LibraryPrivateTypeInPublicAPI())
```

Comment:
Somewhere along the way we (almost/mostly) unified on plural names.  Maybe do the same here?

Reply:
I had the plural form for the lint name, just missed the class name. Done.